features/bevragen/persoon/verblijfplaats/adres/fields.feature

```diff
@@ -0,0 +1,117 @@
+# language: nl
+
+Functionaliteit: Persoon: verblijfplaats binnenland (adres) velden vragen met fields
+
+  Achtergrond:
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'verblijfplaats' gegevens
+    | naam                               | waarde   |
+    | datum aanvang adreshouding (10.30) | 20150808 |
+    | datum ingang geldigheid (85.10)    | 20220222 |
+    | functie adres (10.10)              | W        |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                                       | waarde           |
+    | gemeentecode (92.10)                       | 0518             |
+    | identificatiecode verblijfplaats (11.80)   | 0599010000208579 |
+    | identificatiecode nummeraanduiding (11.90) | 0599200000219678 |
+
+  Abstract Scenario: 'datum aanvang adreshouding (10.30)' wordt gevraagd met field pad '<fields>'
```

Comment:
Waarom een abstract scenario als er maar 1 voorbeeld is. Voorzie je meer voorbeelden?

Reply:
overblijfselen van verkort pad. Wordt aangepast